haskell-api.md

```diff
@@ -117,6 +117,107 @@ possible. Issues include:
 If you are creating a new `PUT` consider if it could be expressed in a more
 granular `PATCH` semantic.
 
+### `PATCH` semantics
+
+Modifications to an existing resource should be made via a `PATCH` request.
+
+### Specific guidelines
+
+A well-formed `PATCH` endpoint
+
+- Should update an entity's `updatedAt` field, if present.
+- Should return an empty response or the modified resource. Ideally, if the
```

Comment:
This this is a Should anyway, why not just say it should return the modified resource? If we have cases that must return empty, consider those taking advantage of this as a should-not-must. Also, returning **modified** resource, implies re-fetching to me, so I would just cut the second sentence myself.

```suggestion
- Should return the modified resource.
```


That said, should any of these be upgraded to a MUST?

Reply:
That's a good point!

Reply:
@pbrisbin how's that look?